cypress/utils/utils.ts

```diff
@@ -93,24 +93,41 @@ let userPassword = Cypress.env("pass");
 const tackleUiUrl = Cypress.env("tackleUrl");
 const { _ } = Cypress;
 
-export function inputText(fieldId: string, text: any): void {
-    cy.get(fieldId).click().focused().clear();
-    cy.wait(200);
-    cy.get(fieldId).clear().type(text);
+export function inputText(fieldId: string, text: any, log = false): void {
+    if (!log) {
+        cy.log(`Type ${text} in ${fieldId}`);
+    }
+    cy.get(fieldId, { log }).click({ log }).focused({ log }).clear({ log });
+    cy.wait(200, { log });
+    cy.get(fieldId, { log }).clear({ log }).type(text, { log });
 }
 
 export function clearInput(fieldID: string): void {
     cy.get(fieldID).clear();
 }
 
-export function clickByText(fieldId: string, buttonText: string, isForced = true): void {
+export function clickByText(
+    fieldId: string,
+    buttonText: string,
+    isForced = true,
+    log = false
+): void {
+    if (!log) {
+        cy.log(`Click by text, id: ${fieldId}, text: ${buttonText}`);
```

Comment:
I feel we don't need to print logs . 
For local debugging we can use 
cy.log() when needed.